docs/reference-metrics-opentelemetry.md

```diff
@@ -478,6 +478,120 @@ The total number of messages dropped from this producer.
     * `wait_for_exclusive`
     * `exclusive_with_fencing`
 
+### Replicator metrics
+
+#### pulsar.broker.replication.message.incoming.count
+The total number of messages received from the remote cluster through this replicator.
+* Type: Counter
+* Unit: `{message}`
+* Attributes:
```

Comment:
seems like these attributes are copied multiple times. can we link a ref, instead?